ChromiumHtmlToPdfLib/FileCache/BasicFileCacheManager.cs

```diff
@@ -6,15 +6,15 @@ namespace ChromiumHtmlToPdfLib.FileCache;
 /// <summary>
 ///     A basic file cache manager that does not use hashing
 /// </summary>
-internal class BasicFileCacheManager : FileCacheManager
+internal class BasicFileCacheManager(string cacheDir, string cacheSubFolder, string policySubFolder) : FileCacheManager(cacheDir, cacheSubFolder, policySubFolder)
```

Comment:
file cache manager factory were refactored to set required fields in constructor to simplify nullability analysis